requirements.txt

```diff
@@ -1,2 +1,2 @@
-fondant==0.7.0
+fondant==0.8.dev4
```

Comment:
Shouldn't we base ourselves on the latest official non-dev release?

Reply:
Yes, this was created before the release. Will update.